fuzzers/072-ordered_wires/Makefile

```diff
@@ -2,8 +2,8 @@
 N := 1
 SPECIMENS := $(addprefix build/specimen_,$(shell seq -f '%03.0f' $(N)))
 SPECIMENS_OK := $(addsuffix /OK,$(SPECIMENS))
-MAX_VIVADO_PROCESS ?= 4
-MAX_PIPS_INSTANCE ?= 340000
+MAX_VIVADO_PROCESS ?= 8
+MAX_PIPS_INSTANCE ?= 680000
```

Comment:
Are you sure about increasing this?  Vivado has some kind of memory leak and/or slowdown as this number gets higher.

Reply:
Actually it is a leftover, I tried to increase this, as locally building the 200T took a huge amount of time (there are sth like 80 milion pips)

Reply:
Something to note is that Kokoro sets `MAX_VIVADO_PROCESS`, https://github.com/SymbiFlow/prjxray/blob/master/.github/kokoro/db-full.sh#L55

Reply:
Fixed, restored as it was previously